ubuntu24.04/Dockerfile

```diff
@@ -67,9 +67,7 @@ ADD install.sh /tmp
 RUN apt-key del 7fa2af80 && OS_ARCH=${TARGETARCH/amd64/x86_64} && OS_ARCH=${OS_ARCH/arm64/sbsa} && \
     apt-key adv --fetch-keys "https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/${OS_ARCH}/3bf863cc.pub"
 
-RUN /tmp/install.sh reposetup && /tmp/install.sh depinstall && \
-    curl -fsSL -o /usr/local/bin/donkey https://github.com/3XX0/donkey/releases/download/v1.1.0/donkey && \
-    chmod +x /usr/local/bin/donkey
+RUN /tmp/install.sh depinstall
```

Comment:
Removing `donkey` as it's no longer needed.

Removing the invocation of the `reposetup` method as the Ubuntu apt repos already come pre-configured with the `12.6.2-cuda-base-ubuntu24.04` base image. Without this change, the logs are spammed with the `Duplicate configuration` warning messages

Reply:
Is the `reposetup` function used anymore?

Reply:
No, I can remove it actually